### ISSUE TITLE
SQLSTATE[HY000]: General error: 7 no connection to the server 

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/db-extractor-pgsql#52]
Previous tables in the job were successfully downloaded. Retry or user exception?

```
[2019-01-17 00:36:16] CRITICAL: PDOException:SQLSTATE[HY000]: General error: 7 no connection to the server 

#0 /code/src/Keboola/DbExtractor/Extractor/PgSQL.php(395): PDO->query('SELECT * FROM i...')
#1 /code/src/Keboola/DbExtractor/Extractor/PgSQL.php(79): Keboola\DbExtractor\Extractor\PgSQL->getTables(Array)
#2 /code/src/Keboola/DbExtractor/Extractor/PgSQL.php(110): Keboola\DbExtractor\Extractor\PgSQL->getColumnMetadataFromTable(Array)
#3 /code/vendor/keboola/db-extractor-common/src/Keboola/DbExtractor/Application.php(166): Keboola\DbExtractor\Extractor\PgSQL->export(Array)
#4 /code/vendor/keboola/db-extractor-common/src/Keboola/DbExtractor/Application.php(65): Keboola\DbExtractor\Application->runAction()
#5 /code/run.php(60): Keboola\DbExtractor\Application->run()
```

https://connection.keboola.com/admin/utils/logs?file=2019/01/17/01/2019-01-17-01-36-17-5c3fce01aa820-exception

https://connection.keboola.com/admin/utils/logs?file=2019/01/17/01/2019-01-17-01-36-17-5c3fce01afedd-log

**Comment by tomasfejfar** [comment]
` CRITICAL: PDOException:SQLSTATE[HY000]: General error: 7 server closed the connection unexpectedly 	This probably means the server terminated abnormally  before or while processing the request.",`

Ref,:

https://keboola.zendesk.com/agent/tickets/9890
https://keboola.zendesk.com/agent/tickets/9889

**Comment by ujovlado** [comment]
Open to iste co spomina Tomas:
https://keboola.slack.com/archives/C09U3R1J4/p1561417352007900

```
PDOException:SQLSTATE[HY000]: General error: 7 server closed the connection unexpectedly \tThis probably means the server terminated abnormally \tbefore or while processing the request. 
```

**Comment by kacurez** [comment]
again
https://keboola.slack.com/archives/C09U3R1J4/p1564441650016700

**Comment by Halama** [comment]
https://keboola.slack.com/archives/C09U3R1J4/p1564700517000100
```
CRITICAL: PDOException:SQLSTATE[HY000]: General error: 7 server closed the connection unexpectedly 	This probably means the server terminated abnormally 	before or while processing the request. {"errFile":"/code/src/Keboola/DbExtractor/Extractor/PgSQL.php","errLine":497,"errCode":"HY000","errTrace":"#0 /code/src/Keboola/DbExtractor/Extractor/PgSQL.php(497): PDO->query('    SELECT \\n   ...')\n#1 /code/vendor/keboola/db-extractor-common/src/Keboola/DbExtractor/Extractor/Extractor.php(347): Keboola\\DbExtractor\\Extractor\\PgSQL->getTables(Array)\n#2 /code/src/Keboola/DbExtractor/Extractor/PgSQL.php(197): Keboola\\DbExtractor\\Extractor\\Extractor->createManifest(Array)\n#3 /code/vendor/keboola/db-extractor-common/src/Keboola/DbExtractor/Application.php(170): Keboola\\DbExtractor\\Extractor\\PgSQL->export(Array)\n#4 /code/vendor/keboola/db-extractor-common/src/Keboola/DbExtractor/Application.php(65): Keboola\\DbExtractor\\Application->runAction()\n#5 /code/run.php(60): Keboola\\DbExtractor\\Application->run()\n#6 {main}","errPrevious":""} []
```

**Comment by pivnicek** [comment]
This will be fixed by #76
